google-cloud-compute-v1/acceptance/google/cloud/compute/v1/test_use_compute_smoke.rb

```diff
@@ -0,0 +1,110 @@
+require 'securerandom'
+require "minitest/autorun"
+
+require "google/cloud/compute/v1/instances"
+require "google/cloud/compute/v1/zone_operations"
+
+$default_zone = 'us-central1-a'
+$default_project = ENV['PROJECT_ID']
+$machine_type = 'zones/'+$default_zone+'/machineTypes/n1-standard-1'
+$image =  'projects/debian-cloud/global/images/family/debian-10'
+
+
+class TestUseComputeSmoke < Minitest::Test
+
+  def setup
+    @client = ::Google::Cloud::Compute::V1::Instances::Rest::Client.new
+    @name = 'rbgapic' + rand(max=10000000).to_s
+    @instances = []
+    if $default_project.eql? NIL
+      skip("PROJECT_ID must be set before running this test")
+    end
+  end
+
+  def teardown
+    @instances.each do |instance|
+      @client.delete(project: $default_project, zone: $default_zone, instance: instance)
+    end
+  end
+
+  def test_create_instance
+    self.insert_resource
+    instance = self.get_instance
+    assert_equal(@name, instance.name)
+    assert_equal(1, (instance.disks).length)
+    assert instance.machine_type.include?($machine_type)
+  end
+
+  def get_instance
+    @client.get(project: $default_project, zone: $default_zone, instance: @name)
+  end
+
+  def insert_resource
+    instance_resource = {
+      name: @name,
+      machine_type: $machine_type,
+      network_interfaces: [{access_configs: [{name: 'default'}]}],
+      disks: [{initialize_params: {source_image: $image}, boot: true, auto_delete: true, type: 'PERSISTENT'}],
```

Comment:
This sort of thing is much more readable if split over multiple lines:

```ruby
      disks: [
        {
          initialize_params: {
            source_image: $image
          },
          boot: true,
          auto_delete: true,
          type: "PERSISTENT"
        }
      ],
```

Reply:
done